plugins/hls-explicit-imports-plugin/src/Ide/Plugin/ExplicitImports.hs

```diff
@@ -187,49 +240,70 @@ instance NFData MinimalImports
 
 type instance RuleResult MinimalImports = MinimalImportsResult
 
-newtype MinimalImportsResult = MinimalImportsResult
-  {getMinimalImportsResult :: [(LImportDecl GhcRn, Maybe T.Text)]}
+data MinimalImportsResult = MinimalImportsResult
+  { forLens        :: [(Range, Int)]
+  , forCodeActions :: RM.RangeMap (Range, Int)
```

Comment:
Why do we need the previous field, can't we just enumerate the `(Range, Int)` pairs from the `RangeMap`?

Reply:
... also why does this have a range in the output? is it the same range as in the input?

Reply:
From what I've researched it's not possible to easily extract the Range key from the RangeMap. Also the range in the output is the range of the textedit, but the range of the input is (in this case) what the client sends over, which is not identical